sockeye/train.py

```diff
@@ -360,7 +396,7 @@ def create_data_iters_and_vocab(args: argparse.Namespace,
             max_seq_len_target=max_seq_len_target,
             bucketing=not args.no_bucketing,
             bucket_width=args.bucket_width)
-        return train_iter, validation_iter, config_data, vocab_source, vocab_target
+        return train_iter, validation_iter, config_data, vocab_source, vocab_target, source_factor_vocabs
```

Comment:
It'd be great to generalize vocab_source to a list of vocabs to avoid the extra return value